gammapy/image/tests/test_lists.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from gammapy.datasets import FermiGalacticCenter
+import numpy as np
+
+
+def test_to_cube():
+    sky_cube_original = FermiGalacticCenter.diffuse_model()
+    image_list = sky_cube_original.to_image_list()
+    assert np.array_equal(image_list.skymaps[0].data, sky_cube_original.data[0])
```

Comment:
Is it possible to just use this here?
```
assert image_list.skymaps[0].data == sky_cube_original.data[0]
```

If no, could you please call [numpy.testing.assert_array_equal](http://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_array_equal.html) instead of `assert np.array_equal`? (that should give a better error message in case it's not the same).